edx_proctoring/templates/proctoring/seq_proctored_exam_ready_to_start.html

```diff
@@ -0,0 +1,62 @@
+{% load i18n %}
+<div class="sequence proctored-exam success-top-bar instructions" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+  <h3>
+    {% blocktrans %}
+      Your Proctoring Installation and Set Up is Complete
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Placeholder content. <strong>Please keep the proctoring software open and running while you complete your exam</strong>.
+    {% endblocktrans %}
+  </p>
+  <div class="proctored-exam-message">
+    <h3>
+    {% blocktrans %}
+      You May Begin Your Exam Now
+    {% endblocktrans %}
+    </h3>
+    <p>
+      {% blocktrans %}
+        Your proctored and timed exam is now ready. Once you start, <strong>you will have {{total_time}} to
```

Comment:
Can we make the changes so that they're reflected in the refreshed sandbox? That way at least we can see them in the sandbox, and @griffresch can make changes if he does not agree with my wording. Thanks.